docs/develop/go/versioning.mdx

```diff
@@ -179,31 +179,23 @@ But we don't want to fail on that change, so we check only the function name par
 
 :::caution
 
-Worker Versioning is currently in Pre-release, and backwards-incompatible changes are coming to the Worker Versioning APIs. For now, you need to provide dynamic configuration parameters to your Temporal Service to enable Worker Versioning:
+Worker Versioning is currently in Pre-release.
 
-```
-temporal server start-dev \
-   --dynamic-config-value frontend.workerVersioningDataAPIs=true \
-   --dynamic-config-value frontend.workerVersioningWorkflowAPIs=true \
-   --dynamic-config-value worker.buildIdScavengerEnabled=true
-```
+See the [Pre-release README.md](https://github.com/temporalio/temporal/blob/main/docs/worker-versioning.md)
+for full information.
```

Comment:
Suggestion:

```
:::tip Support, stability, and dependency info

Worker Versioning is currently in  [Pre-release](/evaluate/release-stages#prerelease).

See the [Pre-release README.md](https://github.com/temporalio/temporal/blob/main/docs/worker-versioning.md) for information about the updates and deprecations related to the now-deprecated Version Set concept and the APIs related to it.

:::
```

Reply:
The README is the new docs, not the old ones.

Reply:
No worries!

The suggestions are 

1. Put this in standard format for release stages (although I don't know if we do pre-release normally. @tomwheeler do we do that?)
2. The first link I suggest is just a definition of the stage
3. The second link is what you put up there "for more information" and is on one line for the sentence.

Reply:
I agree with these suggestions. 

To my understanding, our documentation does not generally cover things at the pre-release stage, but I think Worker Versioning is a special case because documentation on the original implementation was published before those stages were defined (there's a longer story here, but that is the pertinent part).

Reply:
Yep, that's why most of the new docs are in a README rather than on the site.